packages/components/organisms/f-status-banner/CHANGELOG.md

```diff
@@ -3,6 +3,16 @@
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+v1.0.0
```

Comment:
Seems the previous change was 1.0.0 and in package.json this has been bumped to 1.1.0  👀

Reply:
Fixed.